components/modals/integration-info.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <ModalTemplate id="integration-info">
+    <template #content>
+      <label
+        for="integration-info"
+        class="btn btn-sm btn-circle absolute right-2 top-2"
+        >✕</label
+      >
+      <div class="flex space-x-4">
+        <img
+          v-if="getCurrent.provider === 'GOOGLE'"
+          class="h-7 w-7"
+          src="https://www.svgrepo.com/show/475656/google-color.svg"
+          alt=""
+        />
+        <img
+          v-if="getCurrent.provider === 'FACEBOOK'"
+          class="h-7 w-7"
+          src="https://www.svgrepo.com/show/448224/facebook.svg"
+          alt=""
+        />
+        <img
+          v-if="getCurrent.provider === 'GITHUB'"
+          class="h-7 w-7"
+          src="https://www.svgrepo.com/show/512317/github-142.svg"
+          alt=""
+        />
```

Comment:
I think you can just replace v-if with conditional render in src.
For example, if the provider is Google, the src in image tag will render svg accordingly.

```suggestion
                        <img
                          class="h-7 w-7"
                          :src="
                            sso.provider === 'GOOGLE'
                              ? 'https://www.svgrepo.com/show/475656/google-color.svg'
                              : sso.provider === 'FACEBOOK'
                              ? 'https://www.svgrepo.com/show/448224/facebook.svg'
                              : 'https://www.svgrepo.com/show/512317/github-142.svg'
                          "
                          alt=""
                        />
```